dev/com.ibm.ws.concurrent.persistent_fat/fat/src/com/ibm/ws/concurrent/persistent/fat/PersistentExecutorWithFailoverEnabledTest.java

```diff
@@ -0,0 +1,114 @@
+import com.ibm.websphere.simplicity.config.ServerConfiguration;
+
+import componenttest.annotation.Server;
+import componenttest.annotation.TestServlet;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.database.DerbyEmbeddedUtilities;
+import componenttest.topology.impl.LibertyFileManager;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.FATServletClient;
+
+import web.SchedulerFATServlet;
+
+@RunWith(FATRunner.class)
+public class PersistentExecutorWithFailoverEnabledTest extends FATServletClient {
+
+    private static final String APP_NAME = "schedtest";
+    private static final String DB_NAME = "${shared.resource.dir}/data/scheddb";
+
+    private static ServerConfiguration originalConfig;
+
+    @Server("com.ibm.ws.concurrent.persistent.fat")
+    @TestServlet(servlet = SchedulerFATServlet.class, path = APP_NAME)
+    public static LibertyServer server;
+    
+    @ClassRule
+    public static final JdbcDatabaseContainer<?> testContainer = DatabaseContainerFactory.create(DB_NAME);
+
+    /**
+     * Before running any tests, start the server
+     *
+     * @throws Exception
+     */
+    @BeforeClass
+    public static void setUp() throws Exception {
+        // Delete the Derby database that might be used by the persistent scheduled executor and the Derby-only test database
+        Machine machine = server.getMachine();
+        String installRoot = server.getInstallRoot();
+        LibertyFileManager.deleteLibertyDirectoryAndContents(machine, installRoot + "/usr/shared/resources/data/scheddb");
+        LibertyFileManager.deleteLibertyDirectoryAndContents(machine, installRoot + "/usr/shared/resources/data/testdb");
+
+        originalConfig = server.getServerConfiguration();
+        ServerConfiguration config = originalConfig.clone();
+        PersistentExecutor myScheduler = config.getPersistentExecutors().getBy("jndiName", "concurrent/myScheduler");
+        myScheduler.setPollInterval("3h"); // the test case does not expect polling, so set a large value that will never be reached
+        myScheduler.setExtraAttribute("missedTaskThreshold2", "1m"); // TODO rename to missedTaskThreshold and use normal setter if this approach is chosen
+        server.updateServerConfiguration(config);
+
+    	//Get type
+		DatabaseContainerType dbContainerType = DatabaseContainerType.valueOf(testContainer);
+
+    	//Get driver info
+    	String driverName = dbContainerType.getDriverName();
+    	server.addEnvVar("DB_DRIVER", driverName);
+
+    	//Setup server DataSource properties
+    	DatabaseContainerUtil.setupDataSourceProperties(server, testContainer);
+    	
+		//Initialize database
+		DatabaseContainerUtil.initDatabase(testContainer, DB_NAME);
+
+		//Add application to server
+        ShrinkHelper.defaultDropinApp(server, APP_NAME, "web");
+
+        //Create Derby DB if using derby
+        if (dbContainerType == DatabaseContainerType.Derby)
```

Comment:
I do not believe this is necessary.  I see that I had this step in the original `SchedulerTest` class, and I don't think it will hurt anything.  but I might go back and remove it.